frontend/lib/src/components/elements/CodeBlock/styled-components.ts

```diff
@@ -123,21 +123,29 @@ export const StyledCopyButtonContainer = styled.div(({ theme }) => ({
   pointerEvents: "none",
 }))
 
-export const StyledCodeBlock = styled.div(({ theme }) => ({
-  position: "relative",
-  marginLeft: theme.spacing.none,
-  marginRight: theme.spacing.none,
-  marginTop: theme.spacing.none,
-  "&:not(:last-child)": {
-    marginBottom: theme.spacing.lg,
-  },
-
-  "&:hover": {
-    [`${StyledCopyButtonContainer}`]: {
-      opacity: 1,
+export interface StyledCodeBlockProps {
+  /**
+   * The code-block behaves a bit different if it is
+   * used inside markdow.
```

Comment:
super nit: should be `markdown`

Reply:
Argh, is there a (GPT-powered) spellcheck linter 😅